python/python/glide/async_commands/core.py

```diff
@@ -1986,6 +1987,49 @@ async def zremrangebyscore(
             ),
         )
 
+    async def zlexcount(
+        self,
+        key: str,
+        min_lex: Union[InfBound, LexBoundary],
+        max_lex: Union[InfBound, LexBoundary],
+    ) -> int:
+        """
+        Returns the number of members in the sorted set stored at `key` with scores between `min_lex` and `max_lex`.
+
+        See https://redis.io/commands/zlexcount/ for more details.
+
+        Args:
+            key (str): The key of the sorted set.
+            min_lex (Union[InfBound, LexBoundary]): The minimum lex to count from.
```

Comment:
could you change lex = lexicographical order or lexicographic (where or whatever you deem to be appropriate lex doesn't seem to be meaningful). I think it makes more sense. After you have, I'll update the java docs. What do you think?

Reply:
sure. I used lexicographical because that was used elsewhere in the file.